packages/story-editor/src/app/taxonomy/taxonomyProvider.js

```diff
@@ -145,32 +145,21 @@ function TaxonomyProvider(props) {
   );
 
   const addSearchResultsToCache = useCallback(
-    async (
-      taxonomy,
-      {
-        name,
-        // This is the per_page value Gutenberg is using
-        perPage = 20,
-      },
-      addNameToSelection = false
-    ) => {
+    async (taxonomy, args, addNameToSelection = false) => {
       let response = [];
       const termsEndpoint = taxonomy['_links']?.['wp:items']?.[0]?.href;
       if (!termsEndpoint) {
-        return;
+        throw new Error("no href present in taxonomy['_links']['wp:items']");
```

Comment:
💭 Should this crash the app or just log and return?

Reply:
good point